eth-registration/src/main/kotlin/registration/eth/relay/RelayRegistration.kt

```diff
@@ -69,6 +69,29 @@ class RelayRegistration(
         }
     }
 
+    /**
+     * Run a job that every replenishmentPeriod checks that number from config free relays are present. In case of
+     * lack of free relays deploys lacking amount.
+     */
+    fun runRelayReplenishment(): Result<Unit, Exception> {
+        logger.info { "Run relay replenishment" }
+
+        return Result.of {
+            while (true) {
+                logger.info { "Relay replenishment triggered" }
+
+                freeRelayProvider.getRelays().map { relays ->
+                    logger.info { "Free relays: ${relays.size}" }
+                    val toDeploy = relayRegistrationConfig.number - relays.size
+                    if (toDeploy > 0)
+                        deploy(toDeploy, relayRegistrationConfig.ethMasterWallet)
+                }.failure { throw it }
+
+                runBlocking { delay(relayRegistrationConfig.replenishmentPeriod * 1000) }
```

Comment:
Why can't we use `Thread.sleep(relayRegistrationConfig.replenishmentPeriod * 1000) `instead?

Reply:
https://medium.com/@mohakpuri1712/kotlin-coroutines-thread-sleep-vs-delay-63171fe8a24

Reply:
There is explained a difference between Thread.sleep() and delay(). delay() is non-blocking suspend function. However, there is no need to use it.